DevelopGitHub/GitHubExtension/GitHubExtension.Security.DAL/Migrations/201603311144256_Initial migration.cs

```diff
@@ -0,0 +1,145 @@
+namespace GitHubExtension.Security.DAL.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class Initialmigration : DbMigration
```

Comment:
Initial migration? As I remember we already added initial migration....Am I right?